src/main/webapp/WEB-INF/views/cars/drivers/add.jsp

```diff
@@ -0,0 +1,14 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Add driver to car</title>
+</head>
+<body>
+<h1>Please provide information about driver and car</h1>
+<form method="post" action="${pageContext.request.contextPath}/cars/drivers/add">
+    Enter car id: <input type="number" name="car_id">
```

Comment:
what if I input nothing in this field?

Reply:
let's use required attribute for each input